axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -66,6 +66,64 @@
         "api": "https://api.routescan.io/v2/network/testnet/evm/43113/etherscan"
       }
     },
+    "avalanche-fuji": {
+      "name": "Avalanche Fuji",
+      "id": "avalanche-fuji",
+      "axelarId": "avalanche-fuji",
+      "chainId": 43113,
+      "rpc": "https://avalanche-fuji-c-chain-rpc.publicnode.com",
+      "tokenSymbol": "ETH",
+      "confirmations": 1,
+      "gasOptions": {
+        "gasLimit": 5000000
+      },
+      "contracts": {
+        "AxelarGateway": {
```

Comment:
I have deployed gas service for these chains as well (as part of ITS hub deployment). you can add it here for each chain

```
        "AxelarGasService": {
          "collector": "0xba76c6980428A0b10CFC5d8ccb61949677A61233",
          "salt": "AxelarGasService devnet-amplifier",
          "address": "0x1179d44e69ba5252B7478a8602617d5EEeb2F377",
          "implementation": "0xeF805CF590B12FD32F76Eb81A236461Cf4f3D796",
          "deployer": "0xba76c6980428A0b10CFC5d8ccb61949677A61233"
        },
```

`node evm/deploy-upgradable.js -c AxelarGasService -m create2 -n [chain] --args '{"collector": "0xba76c6980428A0b10CFC5d8ccb61949677A61233"}' -s 'AxelarGasService devnet-amplifier'`